### PR TITLE
Add step to go-agent building - GitHub private key.

### PR DESCRIPTION
**Pull request by doctoryes**
An important detail! A detail I forgot when building a new go-agent today...

@edx/pipeline-team Please review.
@edx/devops FYI

### CODE REVIEW

**Review by macdiesel — APPROVED** [review]

**Comment by jzoldak** [comment]
👍

File: docker/build/go-agent/README.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ necessary.
 
 ##Building and Uploading the container to ECS
 
+* Copy the go-agent GitHub private key to this path:
+    - ```docker/build/go-agent/files/go_github_key.pem```
+    - A dummy key is in the repo file.
+    - The actual private key is kept in LastPass - see DevOps for access.
```
Comment (feanil) [inline docker/build/go-agent/README.md:35]: Can you also add a note to not check-in the changes to that key?

**Comment by feanil** [comment]
One nit, then :+1:

File: docker/build/go-agent/README.md (hunk 1 above)
Comment (doctoryes) [reply docker/build/go-agent/README.md:35]: Done.

**Review by feanil — APPROVED** [review]

**Pull request merged**